properties/dev.properties

```diff
@@ -32,15 +32,15 @@ MAPRED_INGEST_OPTS=-useInlineCombiner
 HADOOP_INGEST_OPTS=
 
 #extra CHILD_OPTS (java options)
-CHILD_INGEST_OPTS= 
+CHILD_INGEST_OPTS=
 
 BULK_CHILD_MAP_MAX_MEMORY_MB=2048
 LIVE_CHILD_MAP_MAX_MEMORY_MB=1024
 BULK_CHILD_REDUCE_MAX_MEMORY_MB=2048
 LIVE_CHILD_REDUCE_MAX_MEMORY_MB=1024
 
-BULK_INGEST_DATA_TYPES=mycsv
-LIVE_INGEST_DATA_TYPES=wikipedia
+BULK_INGEST_DATA_TYPES=shardStats
```

Comment:
I think these were intended to stay as mycsv and wikipedia for the quickstart stuff.  Please ask Keith.